modules/acquia_cms_headless/tests/src/Functional/PureHeadlessModeMenuTest.php

```diff
@@ -80,7 +80,7 @@ protected function setUp(): void {
    * @dataProvider providerMenu
    */
   public function testChildMenu(string $selector, string $parentMenuName, array $children): void {
-    if ($this->installModule('acquia_cms_toolbar')) {
+    if ($this->installModule('acquia_cms_toolbar') && $this->installModule('block_content')) {
       $this->drupalGet('/admin/headless/dashboard');
       $page = $this->getSession()->getPage();
```

Comment:
@chandan-singh7929 can we check `/admin/content-models` is also accessible in this test and won't throw any error.

Reply:
Done